code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -142,6 +142,46 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "glass of whiskey"
 	glass_desc = "The silky, smokey whiskey goodness inside the glass makes the drink look very classy."
 	shot_glass_icon_state = "shotglassbrown"
+	var/heal_det_brute_amt = 0.5
+	var/heal_det_burn_amt = 0.5
+
+/datum/reagent/consumable/ethanol/whiskey/on_mob_life(mob/living/carbon/M)
+	if(M.mind && M.mind.assigned_role == "Detective")
+		M.heal_bodypart_damage(heal_det_brute_amt,heal_det_burn_amt) //not a painkiller but it'll have to do
+		. = 1
+	return ..() || .
+
+/datum/reagent/consumable/ethanol/whiskey/kong
+	name = "Kong"
+	description = "Makes You Go Ape!�"
+	color = "#332100" // rgb: 51, 33, 0
+	quality = DRINK_BAD //you don't drink it for the taste
+	addiction_threshold = 30 //min pleasure
```

Comment:
you're never gonna reach this addiction threshold unless you're chugging your bottle

and once you're addicted and run out of your roundstart supply of kong, you can't get more of it, so you can't ever "succumb" to your addiction or relapse

Reply:
I'd recommend making the addiction threshold start much lower and fluctuate like the modafinil OD threshold does. I'd also recommend adding a special addiction effect to this drink, since right now, being addicted to it doesn't do much other than giving you the addiction mood debuff.

Reply:
thanks